### ISSUE TITLE
Cannot pull videos from a course while others work

### ISSUE THREAD

**Issue by ketanshah79** [dgorissen/coursera-dl#151]
Hi there,

I've been using coursera-dl and its been of immense value. Thank you :)

I haven't been able to download assets from this course https://www.coursera.org/learn/competitivestrategy

Not sure if that course is setup differently.

Thanks

**Comment by boethian** [comment]
Agreed. I also cannot download content from this course: https://www.coursera.org/learn/insidetheinternet/

Both of these are "always open" and do not require one to sign up and seem to be structured differently from other courses.

**Comment by OliPelz** [comment]
me to! cannot downlad the insidetheinternet course!

**Comment by StevenMaude** [comment]
There's a pull request for another downloader [here](https://github.com/coursera-dl/coursera/pull/315) that fixes this; seemed to work for me. The "on-demand" courses are handled differently from those with timetabled sessions.
